src/server/index.test.jsx

```diff
@@ -1500,6 +1501,75 @@ describe('Server HTTP Headers - Page Endpoints', () => {
 
     expect(header.vary).toBe('Accept-Encoding');
   });
+
+  it(`should set isUK value to true when 'x-bbc-edge-isuk' is set to 'yes'`, async () => {
+    mockRouteProps({
+      dataResponse: successDataResponse,
+      isAmp: true,
+    });
+
+    await makeRequest('/mundo/articles/c0000000001o', {
+      'x-bbc-edge-isuk': 'yes',
+    });
+
+    expect(renderDocumentSpy).toHaveBeenCalledWith(
+      expect.objectContaining({
+        data: expect.objectContaining({ isUK: true }),
+      }),
+    );
+  });
+
+  it(`should set isUK value to false when 'x-bbc-edge-isuk' is NOT 'no'`, async () => {
+    mockRouteProps({
+      dataResponse: successDataResponse,
+      isAmp: true,
+    });
+    getMvtVaryHeaders.mockReturnValue('mvt-simorgh_dark_mode');
+
+    await makeRequest('/mundo/articles/c0000000001o', {
+      'x-bbc-edge-isuk': 'no',
+    });
+
+    expect(renderDocumentSpy).toHaveBeenCalledWith(
+      expect.objectContaining({
+        data: expect.objectContaining({ isUK: false }),
+      }),
+    );
+  });
+
+  it(`should set isUK value to true when 'x-country' is set to 'gb' and 'x-bbc-edge-isuk' is not available`, async () => {
+    mockRouteProps({
+      dataResponse: successDataResponse,
+      isAmp: true,
+    });
+    getMvtVaryHeaders.mockReturnValue('mvt-simorgh_dark_mode');
+
+    await makeRequest('/mundo/articles/c0000000001o', {
+      'x-country': 'gb',
+    });
+
+    expect(renderDocumentSpy).toHaveBeenCalledWith(
+      expect.objectContaining({
+        data: expect.objectContaining({ isUK: true }),
+      }),
+    );
+  });
+
+  it(`should set isUK value to null when 'x-country' and 'x-bbc-edge-isuk' is not available`, async () => {
+    mockRouteProps({
+      dataResponse: successDataResponse,
+      isAmp: true,
+    });
+    getMvtVaryHeaders.mockReturnValue('mvt-simorgh_dark_mode');
+
+    await makeRequest('/mundo/articles/c0000000001o', {});
+
+    expect(renderDocumentSpy).toHaveBeenCalledWith(
+      expect.objectContaining({
+        data: expect.objectContaining({ isUK: null }),
```

Comment:
Just wondering whether we should set `isUK` to false here rather than null? What's the thinking around setting it to null?

Reply:
We set default to false within the requestContext here: https://github.com/bbc/simorgh/pull/11041/files#diff-cf9f800225e789383232d6d4a4cf51f2e93765040e29d4c9d78dae8ab596fda4R104